src/WrapperApp/components/InputEditor/InputEditorPanel.tsx

```diff
@@ -148,17 +148,19 @@ export default function InputEditorPanel({
 					onChange={(_e, generator) => {
 						if (generator) setGenerator(generator);
 					}}>
-					<ToggleButton
-						color='info'
-						value='local'>
-						Local
-					</ToggleButton>
 					{!demoMode && (
-						<ToggleButton
-							value='remote'
-							color='warning'>
-							Remote
-						</ToggleButton>
+						<>
+							<ToggleButton
+								color='info'
+								value='local'>
+								Local
+							</ToggleButton>
+							<ToggleButton
```

Comment:
can you remove completely these toogle buttons and use only `Local` way of generating input files ?

Reply:
Should I also remove:

- the code responsible for selecting a potentially different converter
- the code responsible for sending a request to a remote converter from src/services/ShSimulatorService.tsx

Reply:
Yes, that would be great !